lib/recording/RecordingManager.js

```diff
@@ -346,7 +359,14 @@ class RecordingManager {
                                 });
                         })
                         .then(() => {
+                            const entryObject = this.recordingList[entryId].entryObject;
                             delete this.recordingList[entryId];
+                            if (resetRecordedEntry && (entryObject.serverType == kalturaTypes.KalturaEntryServerNodeType.LIVE_PRIMARY))
+                            {
+                                recordingEntrySession.updateEntryDuration(true).then(()=>{
+                                    backendClient.resetRecordingEntry(entryObject, false);
```

Comment:
doesn't the reset flag value in call to resetRecordingEntry need be true?
in RecordingEntrySession the call is without flag value meaning BE receives true and creates a new recordedEntryId without wait to unregisterMediaServer and registerMedaiServer calls

Reply:
no
that will cause empty vod entries to be created